definitions/ext-rf_scanner/summary_metrics.yml

```diff
@@ -0,0 +1,48 @@
+msToFirstRead:
+  title: Request Time (ms)
+  
```

Comment:
In summary metrics the `unit` is mandatory, you can use any of the ones from [here](https://github.com/newrelic-experimental/entity-synthesis-definitions/blob/main/docs/summary_metrics.md#metric-unit)